tests/tensor/test_basic.py

```diff
@@ -1667,6 +1671,19 @@ def test_join_matrix_ints(self):
         assert (np.asarray(grad(s.sum(), b).eval()) == 0).all()
         assert (np.asarray(grad(s.sum(), a).eval()) == 0).all()
 
+    def test_join_matrix1_using_column_stack(self):
+        av = np.array([0.1, 0.2, 0.3], dtype="float32")
+        bv = np.array([0.7, 0.8, 0.9], dtype="float32")
+        a = self.shared(av)
+        b = as_tensor_variable(bv)
+        s = column_stack((a, b))
+        want = np.array(
+            [[0.1, 0.7], [0.2, 0.8], [0.3, 0.9]],
+            dtype="float32",
+        )
+        out = self.eval_outputs_and_check_join([s])
+        assert (out == want).all()
```

Comment:
This will also check the shapes match, which is important for these changes
```suggestion
        np.testing.assert_array_equal(out, want, strict=True)
```